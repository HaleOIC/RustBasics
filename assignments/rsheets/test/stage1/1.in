get A1
set A1 3 + 5
get A1
