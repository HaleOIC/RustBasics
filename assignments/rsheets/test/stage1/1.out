A1 = None
A1 = 8
